### ISSUE TITLE
[Feature Request]: Better UI to change between databases

### ISSUE THREAD

**Issue by Racle** [redis/RedisInsight#1857]
**Problem description** (Describe the problem you would like to solve with this feature request or suggested idea).
You can only change database by writing number, this seems little slow.

**How often do you encounter this problem** (Describe how frequently this problem occurs).
Always, I use multiple databases 

**Alternatives considered** (Describe any alternative features or products you've considered).
Even list would be much better choice. 
I would prefefer left side view like in RDM as I can see key count quickly: [image](https://user-images.githubusercontent.com/759091/226842096-3f7bd897-47ed-4622-ae90-218f8138f7c6.png)

**Additional information** (Add any other context or details here).

**Comment by Jimbolino** [comment]
or a selectbox, but please no input field

**Comment by Jimbolino** [comment]
This is how [medis](https://github.com/luin/medis) does it:

<img width="298" alt="image" src="https://user-images.githubusercontent.com/5860587/228577832-a619ac89-7647-4a18-b36b-1f18d2ef9370.png">

**Comment by Zamion101** [comment]
This is also how [Another Redis Desktop Manager](https://github.com/qishibo/AnotherRedisDesktopManager) does. It looks clean like medis.

![image](https://user-images.githubusercontent.com/8071263/232209102-27fc7607-682b-417d-9ba3-62bf67a8a423.png)

Also here some database statistics that can be useful
![image](https://user-images.githubusercontent.com/8071263/232209142-47366f8f-af78-45fa-9264-bb96cd05cec5.png)

**Comment by haitanghuadeng** [comment]
I strongly agree with this advice. The experience of choosing an action is much greater than the experience of input. But to make a supplementary suggestion, this option box can also be entered. Because redis default library number range is 0~255, if useful to dozens of hundreds, the selection box is more troublesome.

**Comment by Maayanbt** [comment]
I'm wondering, why not open two Redisinsight windows for example, with the two dbs opened?
wouldn't that be more comfortable?

**Comment by haitanghuadeng** [comment]
> 我想知道，例如，为什么不打开两个 Redisinsight 窗口，打开两个数据库？那不是更舒服吗？

If you need to view DB0 and DB1, now you need to browse between the two Windows, which is cumbersome. Switching between connected instances rather than database libraries is recommended between multiple applications.

**Comment by Zamion101** [comment]
> Because redis default library number range is 0~255, if useful to dozens of hundreds, the selection box is more troublesome.

Maybe allow users to input as well as select from selectbox? Or maybe filter it when we give input? With that we can select from selectbox and/or input the database we want to open. It's a Win-Win situation in both cases.

**Comment by Jimbolino** [comment]
`INFO KEYSPACE` will list all databases and the number of records in it (default install has only 16 databases)

if the select box shows the number of records (see screenshots above), that would save me a lot of time when i need to find the correct database

**Comment by poychang** [comment]
Select box is needed! When switching back and forward with different projects, this feature is very handy.

**Comment by hotoh** [comment]
Also have this question, why not provide a drop-down box and list which database has data, it would be a good experience

**Comment by hepengju** [comment]
@Racle 

Try [RedisME](https://github.com/hepengju/redis-me)

<img width="900" height="600" alt="Image" src="https://github.com/user-attachments/assets/cfa53eee-1e2b-4cb2-a971-0078f7b5f8d1" />
